src/transformers/models/tapas/tokenization_tapas.py

```diff
@@ -1127,6 +1127,7 @@ def prepare_for_model(
         _, _, num_tokens = self._get_table_boundaries(tokenized_table)
 
         if truncation != TapasTruncationStrategy.DO_NOT_TRUNCATE:
+            self.drop_rows_to_fit = True
```

Comment:
This is OK, we can indeed remove the `drop_rows_to_fit` parameter of `TapasTokenizer`, however we also have a parameter called `cell_trim_length`, which will only have an effect in case the tokenizer is used with `truncation=True`. So we should update its documentation. Currently it says:

"`cell_trim_length` (int, optional, defaults to -1) - If > 0: Trim cells so that the length is <= this value. Also disables further cell trimming, should thus be used with `drop_rows_to_fit` below."

=> we can update this to:
"`cell_trim_length` (int, optional, defaults to -1) - If > 0: Trim cells so that the length is <= this value. Also disables further cell trimming, should thus be used with `truncation` set to to `True`."